extension/src/react/components/App.jsx

```diff
@@ -12,13 +15,13 @@ const App = () => {
         },
         {
             key: "PR",
-            tab: "PRs",
-        }
-    ] 
-    const content = {
-        "Notifs": <h1>Notifications</h1>,
-        "PR": <h1>PRs</h1>
-    }
+            tab: "Pull Requests",
+        },
+        {
+            key: "Analytics",
+            tab: "Analytics",
+        },
+    ];
```

Comment:
You really dont need these keys anymore, u can just have an array of the tab names and use those to check the current tab. Segmented doesnt need to set keys since you can reference the values themselves. This would eradicate your need for extracting the tabs in the SegmentedComponent you created.